hack/jenkins/e2e-runner.sh

```diff
@@ -116,7 +116,7 @@ function get_latest_trusty_image() {
       image="$(gsutil cat "gs://${image_project}/image-indices/latest-test-image-${image_index}")" && break || sleep 1
```

Comment:
To switch from GCI test images to base images, what we need is cat the latest-base-image-${image_index} file instead of latest-test-image-${image_index}, right? If so, why not put such a small change in this PR?